examples/dt/nfv/nfv-ovs-dpdk-sriov-hci/control-plane/service-values.yaml

```diff
@@ -0,0 +1,78 @@
+# local-config: referenced, but not emitted by kustomize
+#
+# CHANGEME:
+#
+# - Change network_vlan_ranges based on EDPM OVS DPDK SRIOV HCI environment.
+# - change numa_nodes in the neutron and neutron_tunnel section
+# based on DPDK NIC's and NUMA alignment in the OVS DPDK SRIOV HCI compute node.
+#
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  preserveJobs: false
+  neutron:
+    customServiceConfig: |
+      [DEFAULT]
+      global_physnet_mtu = 9000
+      [ml2]
+      mechanism_drivers = ovn,sriovnicswitch
+      path_mtu = 0
+      [ovn]
+      vhost_sock_dir = /var/lib/vhost_sockets
+      enable_distributed_floating_ip=False
+      [ml2_type_vlan]
+      network_vlan_ranges = dpdk1:206:209,dpdk2:206:209,sriov1,sriov2 # CHANGEME
+      [ovs]
+      igmp_snooping_enable = True
+      [oslo_policy]
+      policy_file=/etc/neutron/policy.d/policy.yaml
+  ovn:
+    ovnController:
+      nicMappings:
+        datacentre: ospbr
+
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:swift
```

Comment:
Please modify this to not deploy swift and not configure Glance to use it because later (in control-plane.md) the following is run:
```
cd architecture/examples/dt/nfv/nfv-ovs-dpdk-sriov-hci
kustomize build
```
and the above configures glance to use only Ceph RBD and disables Swift.

Reply:
Please replace lines 39-52 with lines 60-74 from here

https://github.com/openstack-k8s-operators/architecture/blob/96c8ded98633f7ebf26a5f9d96909b5ac3025ecd/lib/control-plane/openstackcontrolplane.yaml#L60-L74

Reply:
As in the HCI VA, we set the replica count of glance to 0 and it is later changed to 3 when an actual backend is configured.

Reply:
ok, i will update it. My first idea was to deploy a regular nfv deployment which uses glance and swift (preceph stage) and on top of that update to ceph. As this is not the case any more, this make no sense. I will remove this config about glance

Reply:
thanks, I generated the CR and it looks like this as been addressed by this commit.

https://github.com/openstack-k8s-operators/architecture/compare/a855ddb6cd72d29e39ccc67488c42cbb71efde33..9b3d729ace8391b00c23d668e902f7526a44ea40